frontend/src/components/SessionExpired.vue

```diff
@@ -26,14 +17,23 @@ import { AuthRoutes } from '../utils/constants.js';
 
 import { useAuthStore } from '../store/auth.js';
 
+import AppSimpleCard from './util/AppSimpleCard.vue';
+
 export default {
   name: 'SessionExpired',
-
+  components: {
+    AppSimpleCard
```

Comment:
I'm getting both Prettier and eslint warnings in this file. Lack of trailing comma here and double instead of single quotes on line 34 (same for Logout.vue).

Reply:
Thanks for catching this